pkg/argo_client/manifests.go

```diff
@@ -59,6 +119,16 @@ func GetManifestsLocal(ctx context.Context, argoClient *ArgoClient, name, tempRe
 
 	source := app.Spec.GetSource()
 
+	s := os.Getenv("ECR_LOGIN_ENABLED")
+	ecr_login_enabled, err := strconv.ParseBool(s)
+	if err != nil {
+		log.Fatal()
+	}
+
+	if ecr_login_enabled {
```

Comment:
if you modify the args to include something like `cfg config.ServerConfig`, then you can clean up some of this:

```suggestion
	if cfg.EcrLoginEnabled {
```